samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _bluetooth-mesh-onoff-sample:
+
+Bluetooth: Mesh OnOff & Level Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh node.
+
+Root element has Generic OnOff Server, Generic OnOff Client, Generic Level Server, Generic Level Client models.
+
+Prior to provisioning, an unprovisioned beacon is broadcast that contains
+a unique UUID. It is obtained from the device address set by Nordic in the
+FICR. Each button controls the state of its
+corresponding LED and does not initiate any mesh activity.
+
+Association of Models with h/w (in case of nRF52840-PDK board)
+**************************************************************
+
+LED1 is associated with Gen. OnOFF Server
+
+Button1 & Button2 are associated with Gen. OnOff Client [Button1 : ON & Button 2: OFF]
+
+LED3 is associated with Gen. Level Server [if (Level < 50) LED3: OFF ...if (Level >= 50) LED3: ON]
+
+Button3 & Button4 are associated Gen. Level Client [Button3: publishes Level = 25 .....Button4: publishes Level = 100]
+
+-------------------------------------------------------------------------------------------------------------------------
+
+After provisioning, the button clients must
+be configured to publish and the LED servers to subscribe.
+
+If a LED server is provided with a publish address, it will
+also publish its status on an onoff state change.
+
+Requirements
+************
+
+This sample has been tested on the Nordic nRF52840-PDK board, but would
+likely also run on the nrf52_pca10040 board.
+
+Building and Running
+********************
+
+This sample can be found under :file:`samples/boards/nrf52/mesh/onoff-app` in the
+Zephyr tree.
+
+The following commands build the application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/nrf52/mesh/onoff-app
+   :board: nrf52840_pca10056
+   :goals: build flash
+   :compact:
+
+Prior to provisioning, each button controls its corresponding LED as one
+would expect with an actual switch.
+
+Provisioning is done using the BlueZ meshctl utility. Below is an example that
+binds button 1&2 and LED 1 to application key 1. It then configures button 1&2
```

Comment:
You called these ``Button1`` and ``Button2`` previously (without a space), so I'd be consistent here and write this as:

    Here is an example that binds Button1, Button2, and LED1 to
    application key 1. It then configures Button1 and Button2
    to publish to group 0xc000 and LED1 to subscribe to that group.